map-of-the-web/motw/motw/urls.py

```diff
@@ -0,0 +1,21 @@
+"""motw URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/2.0/topics/http/urls/
+Examples:
```

Comment:
The first line of this commit is great, and the reference to external documentation.

The examples listed here would be better off omitted since this is documentation of Django, not anything specific to our application.